widgets/ScalaSparkCompute-sparkcompute.json

```diff
@@ -23,6 +23,14 @@
             "values" : ["true", "false"],
             "default": "true"
           }
+        },
+        {
```

Comment:
Better follow the same property order as in the ScalaSparkProgram, which is editor, deploy compile option and extra dependencies.